src/_includes/components/site-header.html

```diff
@@ -1,4 +1,4 @@
-<header>
-  {% include components/site-logo.html %}
+<header id="nav-header">
```

Comment:
What is/was your reasoning for using an id here?  In CSS ids are top specificity and do not lend themselves well to extensibility. As a general rule, don't use ids. There is no need, especially with modern js querySelector.

Reply:
`.site-header` would be more appropriate. This component is more than just the `nav` so the id would not be semantic.

Reply:
I saw it as an element that would only be used once per page, with styling specific to the nav, so I considered it safe to use an ID.